### PR TITLE
Change postgres version from 11.10 to 11.15

### PR DESCRIPTION
**Pull request by fhambrec**
The latest minor version of postgresql v11 is 15. This pull request bumps the postgres version of this bosh release from 11.10 to 11.15 and exchanges all related spec/package files and folders. To enable the usage of the new release the [postgres11.15-archive](https://www.postgresql.org/ftp/source/v11.15/) needs to be added to the official blobstore.

### CODE REVIEW

**Review by beyhan — CHANGES_REQUESTED** [review]
Overall the change looks good. What I found is that the [versions.yml](https://github.com/cloudfoundry/postgres-release/blob/develop/versions.yml) needs to be updated and see my other comment in the blobls.yml.

File: config/blobs.yml [hunk 1]
```diff
@@ -6,3 +6,6 @@ postgres/postgresql-11.10.tar.gz:
   size: 26206541
```
[the remark this answers is not in the document]
Comment (fhambrec) [reply config/blobs.yml]: We removed the unused blob. Regarding the versions.yml: We noticed that the versions.yml is being taken care of from the `create-final-release` task/pipeline. Is this pipeline part of some release process that is triggered after a successful merge? If so, who is responsible for that pipeline?

[the remark this answers is not in the document]
Comment (beyhan) [reply config/blobs.yml]: I'm not familiar how the release setup is done for this project. @rkoster do you know? It looks to me that:

- we shouldn't change `blobs.yml` manually. This should happen during the `bosh upload-blobs`. Sorry, I missed this during the first review
- the blob for the new release needs to uploaded manually
- the [create-final-release](https://github.com/cloudfoundry/postgres-release/blob/e300190594565d0e21b39c744990b212e224b3d3/ci/scripts/create-final-release/task.sh) task will create the final release and add its version to the `versions.yml` but doesn't upload any blobs.
- The [old](https://github.com/cloudfoundry/postgres-release/blob/e300190594565d0e21b39c744990b212e224b3d3/versions.yml#L45) part in the `versions.yml` is maintained manually.

**Review by beyhan — APPROVED** [review]

**Comment by rkoster** [comment]
Thanks! @fhambrec

**Review by rkoster — COMMENTED** [review]
LGTM

**Pull request merged**